### ISSUE TITLE
Links should comply with WCAG AA guidelines

### ISSUE THREAD

**Issue by rbnch** [keepandroidopen/keepandroidopen.github.io#46]
Per [WCAG success criterion 1.4.11](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html):
> The visual presentation of the following have a contrast ratio of at least 3:1 against adjacent color(s):
> ...
> Visual information required to identify user interface components and states, except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author

This site's body text color `#24292e`, so link color should be [at least `#036bea`](https://webaim.org/resources/linkcontrastchecker/?fcolor=24292E&bcolor=FFFFFF&lcolor=036BEA).

Or you can adjust the body text color ([`#1f2127` is the minimum compliant color](https://webaim.org/resources/linkcontrastchecker/?fcolor=1F2127&bcolor=FFFFFF&lcolor=0366DA)).
